x-pack/plugins/apm/public/components/shared/Links/url_helpers.ts

```diff
@@ -23,7 +23,10 @@ export function fromQuery(query: APMQueryParams) {
     if (['rangeFrom', 'rangeTo'].includes(key!)) {
       return value;
     }
-    return qs.escape(value.toString());
+    if (value === undefined) {
```

Comment:
It seems like `value` is any - else this wouldn't compile. Since we know all values of `APMQueryParams` can we make Typescript catch this?

Reply:
Yeah good point, I am not sure why it didn't already catch it. I guess mapValues isn't smart enough? I'll see what we can do.

Reply:
Actually ... now that #32732 fixes this problem, I think we might want to just revert all of this back to what it was yesterday.

Reply:
Yeah, if that works I'm good with that